configstack/module.go

```diff
@@ -83,7 +84,18 @@ func resolveTerraformModule(terragruntConfigPath string, terragruntOptions *opti
 		return nil, err
 	}
 
-	return &TerraformModule{Path: modulePath, Config: *terragruntConfig, TerragruntOptions: opts}, nil
+	if terragruntConfig.Terraform != nil {
+		s := []string{filepath.Dir(terragruntConfigPath), "*.tf"}
```

Comment:
Also, instead of using an array here and then joining it below, try:

```go
matches, err := filepath.Glob(filepath.Join(filepath.Dir(terragruntConfigPath), "*.tf"))
```